Python/Product/Uwp/Project/PythonUwpProjectConfig.cs

```diff
@@ -711,6 +684,79 @@ class PythonUwpProjectConfig :
             return result;
         }
 
+        private void OnBootstrapEnd(string projectUniqueName, IVsAppContainerBootstrapperResult result) {
+            if (result == null || !result.Succeeded) {
+                this.NotifyEndDeploy(0);
+                return;
+            }
+
+            IVsDebuggerDeploy deploy = (IVsDebuggerDeploy)Package.GetGlobalService(typeof(SVsShellDebugger));
+            IVsDebuggerDeployConnection deployConnection;
+
+            int hr = deploy.ConnectToTargetComputer(result.Address, VsDebugRemoteAuthenticationMode.VSAUTHMODE_None, out deployConnection);
+            lock (syncObject) {
+                connection = deployConnection;
+            }
+
+            if (ErrorHandler.Failed(hr)) {
+                NotifyEndDeploy(0);
+                return;
+            }
+
+            string recipeFile = GetRecipeFile();
+            IVsAppContainerProjectDeploy deployHelper = (IVsAppContainerProjectDeploy)Package.GetGlobalService(typeof(SVsAppContainerProjectDeploy));
+            uint deployFlags = (uint)(_AppContainerDeployOptions.ACDO_NetworkLoopbackEnable | _AppContainerDeployOptions.ACDO_SetNetworkLoopback);
+            IVsAppContainerProjectDeployOperation localAppContainerDeployOperation = deployHelper.StartRemoteDeployAsync(deployFlags, connection, recipeFile, projectUniqueName, this);
+
+            lock (syncObject) {
+                this.deployOp = localAppContainerDeployOperation;
+            }
+        }
+
+        private IVsAppContainerBootstrapperResult BootstrapForDebuggingSync(string targetDevice) {
+            IVsAppContainerBootstrapper4 bootstrapper = (IVsAppContainerBootstrapper4)ServiceProvider.GlobalProvider.GetService(typeof(SVsAppContainerProjectDeploy));
+            return (IVsAppContainerBootstrapperResult)bootstrapper.BootstrapForDebuggingAsync(this.GetProjectUniqueName(), targetDevice, BootstrapMode.UniversalBootstrapMode, this.GetRecipeFile(), logger: null).GetResult();
+        }
+
+        private string GetProjectUniqueName() {
+            EnvDTE.DTE dte = (EnvDTE.DTE)Package.GetGlobalService(typeof(EnvDTE.DTE));
+            EnvDTE.SolutionBuild sb = dte.Solution.SolutionBuild;
+            string projectUniqueName = string.Empty;
+            foreach (String s in (Array)sb.StartupProjects) {
+                // There should only be one startup project so get the first one we find.
```

Comment:
That's not true - you can set multiple startup projects in VS (right-click on solution node -> "Set Startup Projects").